libs/parallelism/algorithms/include/hpx/parallel/datapar/iterator_helpers.hpp

```diff
@@ -490,6 +528,44 @@ namespace hpx { namespace parallel { namespace util { namespace detail {
             }
         }
     };
+
+    struct datapar_transform_loop_step_ind
+    {
+        template <typename F, typename InIter, typename OutIter>
+        HPX_HOST_DEVICE HPX_FORCEINLINE static void call1(
+            F&& f, InIter& it, OutIter& dest)
+        {
+            typedef
+                typename std::iterator_traits<InIter>::value_type value_type;
+
+            typedef typename traits::vector_pack_type<value_type, 1>::type V1;
+
+            invoke_vectorized_inout1_ind<V1>::call_aligned(
+                std::forward<F>(f), it, dest);
```

Comment:
Should this be `call_unaligned`?

Reply:
@hkaiser yes, thanks! i would change it in next commit! ;-)